WORKSPACE

```diff
@@ -19,15 +19,20 @@ http_archive(
     urls = ["https://github.com/rules-proto-grpc/rules_proto_grpc/archive/3.1.1.tar.gz"],
 )
 
-load("@rules_proto_grpc//:repositories.bzl", "rules_proto_grpc_toolchains", "rules_proto_grpc_repos")
+load("@rules_proto_grpc//:repositories.bzl", "rules_proto_grpc_repos", "rules_proto_grpc_toolchains")
+
```

Comment:
nit/clarification: were these blank lines added by `buildifier`?

Reply:
Yes, I tried deleting them and re-running it and they got re-added.